qa/ccs-unavailable-clusters/src/javaRestTest/java/org/elasticsearch/search/CrossClusterSearchUnavailableClusterIT.java

```diff
@@ -182,97 +182,223 @@ public void testSearchSkipUnavailable() throws IOException {
             assertEquals(200, refreshResponse.getStatusLine().getStatusCode());
 
             {
-                SearchResponse response = restHighLevelClient.search(new SearchRequest("index"), RequestOptions.DEFAULT);
-                assertSame(SearchResponse.Clusters.EMPTY, response.getClusters());
-                assertEquals(10, response.getHits().getTotalHits().value);
-                assertEquals(10, response.getHits().getHits().length);
+                SearchResponse searchResponseIndex = null;
```

Comment:
I don't think this is the right solution here. I open https://github.com/elastic/elasticsearch/pull/102049 to remove the high level rest client all together.

Reply:
Thanks for handling this @iverase ! Reverting in favor of #102049 .